crates/nu-command/src/conversions/into/duration.rs

```diff
@@ -324,7 +363,7 @@ fn convert_str_from_unit_to_unit(
             dst_span: span,
             src_span: value_span,
             help: Some(
-                "supported units are ns, us, ms, sec, min, hr, day, wk, month, yr and dec"
+                "supported units are ns, us, µs, ms, sec, min, hr, day, wk, month, yr and dec"
```

Comment:
Maybe a style nit from me but I would differentiate that from the enumeration. And Oxford commas all the way...

```suggestion
                "supported units are ns, us/µs, ms, sec, min, hr, day, wk, month, yr, and dec"
```